seeds/nft/ethereum/nft_ethereum_loans_seed.csv

```diff
@@ -0,0 +1,106 @@
+evt_tx_hash,evt_block_time,borrower,lender,collectionContract,tokenId,apr,duration,source,eth,usd,status,currency,principal_raw,collectionName
+0x518ed2220373d853568ccff6a6ec93078eddf0d01dee3f637bcbe06df49c1097,2022-01-27T11:20:14Z,0x51981370e2ca51bc8ff6fa96f3c272e1cfb00bc9,0x0aff497bd016000185b1c8302fa98a88ff4a4178,0x50f5474724e0ee42d9a4e711ccfb275809fd6d4a,156440,,14.0,NFTFi,,,REPAID,0x6b175474e89094c44da98b954eedeac495271d0f,,Sandbox's LANDs
+0x99d881f7284a70e6ebed0685c0f3a3a165a2c111e91796513f5a651c000f45d7,2021-11-19T11:42:59Z,0xb149f353383ab92e74c4cba3feb848b04bfb3617,0x0b7059d9e71bbd6e28677386b4c13c56b74b944d,0x5ab21ec0bfa0b29545230395e3adaca7d552c948,9504,,30.0,NFTFi,,,REPAID,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,,PUNKS Comic
+0xfe671ecda91324cb7694501e3e1290454af06f0992a9deccd2b0a4cacce85f29,2021-07-28T20:03:32Z,0xc70b0b44e47e8604b4234f2d2d8e79540b0cf64b,0x6c3e007377effd74afe237ce3b0aeef969b63c91,0xa7d8d9ef8d8ce8992df33d8b8cf4aebabd5bd270,78000176,,90.0,NFTFi,,,REPAID,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,,Art Blocks Curated
+0x8c85ccd367de2d4d2ac5f97147666e77095db1d1d16db4a37d9b42062e38b5ff,2021-12-31T20:20:14Z,0xf02cd6f7b3d001b3f81e747e73a06ad73cbd5e5b,0x312c5f72dc999624eedaf03f67640c29b2916bfd,0xc0cf5b82ae2352303b2ea02c3be88e23f2594171,22700082637,,14.0,NFTFi,,,REPAID,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,,The Fungible by Pak

```

Comment:
I think the failed tests may be cased by the empty/null values in `principal_raw` field.

Reply:
Thanks. This is very useful. Didn't know how to the debug this.